pkg/didcomm/transport/ws/upgrade_js_wasm.go

```diff
@@ -30,3 +31,7 @@ func acceptRecipient(pool *connPool, keys []string) bool {
 
 	return false
 }
+
+func keepConnAlive(conn *websocket.Conn, outbound bool, frequency time.Duration) {
+	// TODO make sure connection is alive (conn.Ping() doesn't work with JS/WASM build)
```

Comment:
So the WASM won’t work?

Reply:
I'm wondering if this needs to be handled at the proxy server/load balancer.  Lots of browser based applications use websocket and don't think they would need to reconnect every min. I'll do more reading on this.

Reply:
Ok